zubhub_frontend/zubhub/src/assets/js/styles/views/create_project/createProjectStyles.js

```diff
@@ -66,6 +67,23 @@ const styles = theme => ({
       },
     },
   },
+  customBoxStyle:{
```

Comment:
Multiple comments on this, use appropriate name like circleBox. For colors use theme